### ISSUE TITLE
[Question] What hacks this mod has?

### ISSUE THREAD

**Issue by ghost** [RealDebugMonkey/ZeachCobbler#62]
Hello,
on heyitsleo's Agar.io bot page in other projects, he wrote that Zeach Cobbler Bot has hacks.
What exactly hacks this mod has?

**Comment by RealDebugMonkey** [comment]
Hahaha. Depends on your definition of 'hacks' but it means the same thing that's in the Readme for the project: `*Please note that the Agario community at large considers many of the features contained in this mod hacks or cheats as it could be argued that they give you an unfair advantage over others.*`
This means the grazer bot, that lite-brite mode lets you see under viruses, the virus popper, etc.

In other words, changes to the underlying client-side code that give people an unfair advantage. But **_not**_ hacks as in doing bad stuff to the server.

**Comment by ghost** [comment]
@RealDebugMonkey thanks, I just wonder how it was meant :+1:
